modules/apps/frontend-editor/test.properties

```diff
@@ -1,3 +1,9 @@
+##
+## Playwright
+##
+
+    playwright.test.project[playwright-js-tomcat90-mysql57-jdk8][relevant][frontend-editor-rule]=frontend-editor-rule
+
```

Comment:
Ooh what I said in https://github.com/liferay-frontend/liferay-portal/pull/4394#discussion_r1740598558 is not true anymore, `test.properties` can and should be on module level. That's related to relevant rules engine changes they [talked about in Slack recently](https://liferay.slack.com/archives/C19PWQ74J/p1722901714828719).

So, as the SF error says, `frontend-editor-ckeditor-sample-web` is missing the `test.properties` file. It will me something like for [frontend-data-set-admin module](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-admin-web/test.properties), with no special logic. Just different module name in file content.

The change above, on app level, is not needed.